### PR TITLE
adding a link to the integrations catalog  release page 

### PR DESCRIPTION
**Pull request by YANG-DB**
### Description
adding a link to the integrations catalog  release page in the main integration page
integration-main-page

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by rupal-bq — APPROVED** [review]

File: public/components/integrations/components/integration_header.tsx [hunk 1]
```diff
@@ -67,6 +70,13 @@ export function IntegrationHeader() {
           Learn more
         </EuiLink>
       </EuiText>
+      <EuiSpacer size="s" />
+      <EuiText size="s" color="subdued">
+        Integrations release catalog page.{' '}
+        <EuiLink external={true} href={OPENSEARCH_INTEGRATION_CATALOG_URL} target="blank">
+          Catalog Page Link
```
Comment (derek-ho) [inline public/components/integrations/components/integration_header.tsx:77]:
```suggestion
          OpenSearch Catalog
```

I feel like page and link are redundant here, do we need UX review of these texts?

**Comment by Swiddis** [comment]
My instinct is that this would be better placed as a button in the right, as a sort-of supplementary action, instead of in the description. Something like "View Catalog" with a hover-tooltip explaining something along the lines of "The Catalog is where new integration updates are released, which you can import to your cluster at any time"

Screenshot 2024-06-03 at 10 46 52 AM

**Review by Swiddis — CHANGES_REQUESTED** [review]
Failing unit tests (snapshots)

**Comment by Swiddis** [comment]
Discussed with author and UX team: I'll be implementing the above button solution along with the upload button mentioned in #1867, closing this PR in favor of the new one.

**Pull request closed without merging**